src/test/java/com/puppycrawl/tools/checkstyle/CheckerTest.java

```diff
@@ -1424,12 +1422,12 @@ public void testTabViolation() throws Exception {
             createModuleConfig(VerifyPositionAfterTabFileSet.class);
         final DefaultConfiguration checkerConfig = createRootConfig(checkConfig);
         checkerConfig.addProperty("tabWidth", "4");
-        final String[] expected = {
-            "2:5: violation",
-            "3:9: violation",
-        };
-        verify(checkerConfig, getPath("InputCheckerTabCharacter.txt"),
-            expected);
+
+        execute(checkerConfig, getPath("InputCheckerTabCharacter.txt"));
+
+        assertWithMessage("Tab width should be set to 4")
+            .that(checkerConfig.getProperty("tabWidth"))
```

Comment:
Please explain why not a verify

Reply:
execute method covered this below line of setTabWidth method

`this.tabWidth = tabWidth;`

```
/**
     * Set the tab width to report audit events with.
     *
     * @param tabWidth an {@code int} value
     */
    public final void setTabWidth(int tabWidth) {
        this.tabWidth = tabWidth;
    }
```

here is the surviving mutation which i want to cover through this test so that pitest common get pass

```
<mutation unstable="false">
+    <sourceFile>Checker.java</sourceFile>
+    <mutatedClass>com.puppycrawl.tools.checkstyle.Checker</mutatedClass>
+    <mutatedMethod>setTabWidth</mutatedMethod>
+    <mutator>org.pitest.mutationtest.engine.gregor.mutators.experimental.MemberVariableMutator</mutator>
+    <description>Removed assignment to member variable tabWidth</description>
+    <lineContent>this.tabWidth = tabWidth;</lineContent>
+  </mutation>
```


while using verifyWithInlineConfigParser is not covering that same line (this.tabWidth = tabWidth;)
this was the first reason

second reason i found that is, a repeatation of same test method is i use verifyWithInlineConfigParser in **testTabViolation** method. see below

```
@Test
    public void testTabViolationDefault() throws Exception {
        final String[] expected = {
            "10:17: violation",
            "13:33: violation",
        };
        verifyWithInlineConfigParser(getPath("InputCheckerTabCharacter.txt"),
            expected);
    }
    
    @Test
    public void testTabViolation() throws Exception {
        final String[] expected = {
            "10:17: violation",
            "13:33: violation",
        };
        verifyWithInlineConfigParser(getPath("InputCheckerTabCharacter.txt"),
            expected);
    }
```

may be i am wrong, not very much sure about analysis.

but pitest is still failing, 
i need to figure out that.

Reply:
Not a good reason.
Please do update like at https://github.com/checkstyle/checkstyle/pull/14462 to cover coverage of Treewalker